storm-server/src/main/java/org/apache/storm/daemon/supervisor/ReadClusterState.java

```diff
@@ -69,10 +65,8 @@
     public ReadClusterState(Supervisor supervisor) throws Exception {
         this.superConf = supervisor.getConf();
         this.stormClusterState = supervisor.getStormClusterState();
-        this.syncSupEventManager = supervisor.getEventManger();
-        this.assignmentVersions = new AtomicReference<>(new HashMap<>());
         this.assignmentId = supervisor.getAssignmentId();
-        this.iSuper = supervisor.getiSupervisor();
+        this.iSupervisor = supervisor.getiSupervisor();
```

Comment:
Please rebase the change with latest master and fix new code with `iSuper`. The compilation breaks with latest master + current patch.

Reply:
done